chart/README.md

```diff
@@ -134,7 +134,7 @@ helm upgrade airflow . \
 
 ## Mounting DAGS using Git-Sync side car without Persistence
 
-This option will use an always running Git-Sync side car on every scheduler,webserver and worker pods. The Git-Sync side car containers will sync DAGs from a git repository every configured number of seconds. If you are using the KubernetesExecutor, Git-sync will run as an initContainer on your worker pods.
+This option will use an always running Git-Sync side car on every scheduler, webserver and worker pods. The Git-Sync side car containers will sync DAGs from a git repository every configured number of seconds. If you are using the KubernetesExecutor, Git-sync will run as an initContainer on your worker pods.
```

Comment:
This is a simple typo fix. Not related to the scope of this PR. Not worth creating a separate PR.